mobile/src/main/java/org/hawkular/client/android/fragment/TriggersFragment.java

```diff
@@ -100,6 +100,15 @@ public void onActivityCreated(Bundle state) {
         setUpTriggers();
     }
 
+    @Override public void onResume() {
```

Comment:
Sorry I think there is some issue with Android studio on my laptop.
I will make sure that it doesn't happen again